webdriver/tests/support/helpers.py

```diff
@@ -23,6 +23,11 @@ def inner(*args, **kwargs):
 
 def cleanup_session(session):
     """Clean-up the current session for a clean state."""
+
+    # Do not try to clean up already ended session.
+    if session.session_id is None:
+        return
+
```

Comment:
Lets move those lines down right before we actually call the individual helper methods. Otherwise we have their definitions in the middle of the executing code.